### ISSUE TITLE
A11y: QuickView details button focus indicator has insufficient contrast

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#5267]
The focus indicator for the show more details button has insufficient contrast against both the button color AND the background color.

{panel:bgColor=#deebff}
The coloring for this also applies to the cart overlay, so we need to make sure we use the same as storefront.
{panel}

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **CMS-481**



[created from CMS-481, comment 506640]

**Comment by shopwareBot** [comment]
[public] The issue is already covered by the following ticket: [https://github.com/shopware/shopware/issues/5206](https://github.com/shopware/shopware/issues/5206)



[created from CMS-481, comment 518709]
